app/src/main/java/com/animal_villa/dto/Character.kt

```diff
@@ -1,4 +1,7 @@
 package com.animal_villa.dto
 
-class Character {
+data class Character(var name : String, var occupation : String)    {
+    override fun toString(): String {
+        return "$name $occupation"
+    }
 }
```

Comment:
Data class for character. Same as Animal, but it is what i'll be using for the TDD example. https://kotlinlang.org/docs/data-classes.html